project-base/storefront/components/Basic/GoogleMap/GoogleMap.tsx

```diff
@@ -0,0 +1,142 @@
+import { GoogleMapMarker } from './GoogleMapMarker';
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import GoogleMapReact from 'google-map-react';
+import getConfig from 'next/config';
+import { useEffect, useMemo, useState } from 'react';
+import useSupercluster from 'use-supercluster';
+
+type GoogleMapMarker = {
```

Comment:
What do we need this type for? I can see that it is a store type from the outside, but here you define a special type for it.

Reply:
Removed.